decide-app/src/test/java/com/decide/app/DecideTest.java

```diff
@@ -9,10 +9,12 @@ public class DecideTest
     @Test
     public void shouldAnswerWithTrue()
     {
-        int NUMPOINTS = 1;
-        Point[] POINTS = new Point[]{ new Point(1, 2) };
+        int NUMPOINTS = 3;
+        Point[] POINTS = new Point[]{ new Point(1, 2),new Point(-1, 2),new Point(1, -2) };
         Parameters PARAMETERS = new Parameters();
         PARAMETERS.AREA1 = 1.0;
+        PARAMETERS.QUADS = 1;

```

Comment:
Remove this and add it to the default constructor in Parameters instead (set QUADS = 1). More exists on the main branch and will be handled when you merge.